CHANGELOG.md

```diff
@@ -22,6 +22,14 @@
   * Simply use `any` as the verb when you define it (i.e. any '/things/:id' )
 * Allow a MediaType to define a custom `links` attribute like any other.
   * This is not compatible if it also wants to use the `links` DSL.
+* First pass at describing (and doc-generating) API global information
+  * Inside a `Praxis::ApiDefinition.define` block one can now specify a few things about the API by using:
+    * info("1.0") `<block>` - Which will apply to a particular version only
+    * info `<block>` - Which will be inherited by any existing API version
+    * The current pieces of information that can be defined in the block are: `name`, `title`, `description` and `basepath`. See [this](https://github.com/rightscale/praxis/blob/master/spec/spec_app/design/api.rb) for details
```

Comment:
Might want to describe the intended use and difference between `name` and `title`, as well as that both are required.